collector/collector.go

```diff
@@ -365,7 +364,7 @@ func (c Collector) collect(ch chan<- prometheus.Metric, logger log.Logger, clien
 	results, err := ScrapeTarget(client, c.target, c.auth, module.Module, logger, c.metrics)
 	c.metrics.SNMPInflight.Dec()
 	if err != nil {
-		level.Info(logger).Log("msg", "Error scraping target", "err", err)
+		logger.Info("Error scraping target", "err", err)
```

Comment:
There are several instances like this that are logging errors at levels other than `error` -- is this intentional?

Reply:
I will need to check the history, but these probably should be `Debug()` or `Warn()`.

Reply:
I went through the history of the Info logging in `collector.go` and it seems like these were recent logging level mistakes.

For now let's preserve the existing logging levels and we can revisit them later in a separate PR.